packages/switchToggle/src/Default/Default.types.ts

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface SwitchToggleContainerProps {
```

Comment:
Once you are intersecting the `bs.FormCheckProps` at 19, you don't need to declare this `SwitchToggleContainerProps`.